process_report/tests/unit/test_chromium.py

```diff
@@ -0,0 +1,63 @@
+import os
+import shutil
+import tempfile
+from unittest import TestCase
+
+import pandas as pd
+
+from process_report.invoices.pi_specific_invoice import PIInvoice
+from process_report.invoices import invoice
+
+
+class TestPIInvoiceExport(TestCase):
+    def setUp(self):
```

Comment:
You've already added a [base test class](https://github.com/CCI-MOC/invoicing/blob/main/process_report/tests/base.py) that provides a temp dir. Please use that to avoid code repetition.